extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/ForwardedParser.java

```diff
@@ -156,7 +183,7 @@ private void calculate() {
 
         host = host + (port >= 0 ? ":" + port : "");
         delegate.headers().set(HttpHeaders.HOST, host);
-        absoluteURI = scheme + "://" + host + delegate.uri();
+        absoluteURI = scheme + "://" + host + uri;
```

Comment:
little effort, big effect for later debugging: please log the absoluteURI:
`log.debug("recalculated absoluteURI to " + absoluteURI);`

Reply:
done!